### ISSUE TITLE
Permission error installing plugin

### ISSUE THREAD

**Issue by iorbitearth** [fedej/poetry-grpc-plugin#1]
Wondering if the error is expected in this early phase of development or something specific to environments that can be corrected by users:

```
$ poetry plugin add poetry-grpc-plugin
Using version ^0.1.0 for poetry-grpc-plugin

  PermissionError

  [Errno 13] Permission denied: '/usr/pyproject.toml'

  at /usr/lib/python3.9/pathlib.py:1110 in _opener
      1106│         pass
      1107│ 
      1108│     def _opener(self, name, flags, mode=0o666):
      1109│         # A stub for the opener argument to built-in open()
    → 1110│         return self._accessor.open(self, flags, mode)
      1111│ 
      1112│     def _raw_open(self, flags, mode=0o777):
      1113│         """
      1114│         Open the file pointed by this path and return a file descriptor,
```

```
$ file /usr/pyproject.toml
/usr/pyproject.toml: cannot open `/usr/pyproject.toml' (No such file or directory)
```

```
$ poetry --version
Poetry (version 1.2.0a2)
```

```
$ python --version
Python 3.9.6
```

**Comment by iorbitearth** [comment]
This seems to be an issue with poetry preview version 1.2.0a2 (or at least on my environment). Same issue installing other plugins as well.

**Comment by iorbitearth** [comment]
Confirmed this issue is resolved when installing the preview (1.2.0a2) version of poetry using python3.8

```
curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3.8 - --preview
```
